vnext/src-win/Libraries/Components/Button.windows.js

```diff
@@ -576,7 +576,7 @@ const styles = StyleSheet.create({
   },
   buttonPressed: {
     borderColor: PlatformColor('ButtonBorderBrushPressed'),
-    borderBottomWidth: 1,
+    borderBottomWidth: 1.5,
```

Comment:
So this is intended to match what's on line 521?
If these are layered together, is it the same to remove this prop override entirely?

Reply:
I removed the override in the latest commit. Thank you.